dev/com.ibm.ws.security.spnego_fat.1/fat/src/com/ibm/ws/security/spnego/fat/FATSuite.java

```diff
@@ -0,0 +1,164 @@
+/*******************************************************************************
+ * Copyright (c) 2014, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.spnego.fat;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.ClassRule;
+import org.junit.rules.ExternalResource;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.security.spnego.fat.config.CommonTest;
+import com.ibm.ws.security.spnego.fat.config.InitClass;
+import com.ibm.ws.security.spnego.fat.config.SPNEGOConstants;
+
+import componenttest.rules.repeater.JakartaEE9Action;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.JavaInfo;
+import componenttest.topology.impl.JavaInfo.Vendor;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+                DynamicSpnegoConfigTest.class,
+                S4U2SelfTest.class,
+                S4U2ProxyTest.class
+})
+public class FATSuite extends InitClass {
+    private static final Class<?> c = FATSuite.class;
+
+    @ClassRule
+    public static RepeatTests repeat = RepeatTests.withoutModification().andWith(new JakartaEE9Action());
```

Comment:
Do you plan to only run the Jakarta 9 ones with lite mode?

Reply:
Typically when I move these over, I set them to run EE9 only in lite, and everything in full. However; if I recall correctly these were written so that pretty much everything is run in full. I didn't change that, so making it so that EE9 only ran in lite really had no consequence.